src/app/components/sandbox/Preview/index.js

```diff
@@ -158,7 +166,8 @@ export default class Preview extends React.PureComponent {
     const element = document.getElementById('sandbox');
 
     if (element) {
-      element.contentWindow.postMessage(message, frameUrl());
+      const { urlInAddressBar: url } = this.state
+      element.contentWindow.postMessage(message, frameUrl(url));
```

Comment:
I don't think this needs to have the url, it's a check for targetOrigin (https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage).